Lib/test/test_zipimport.py

```diff
@@ -37,14 +37,10 @@ def get_file():
 
 def make_pyc(co, mtime, size):
     data = marshal.dumps(co)
-    if type(mtime) is type(0.0):
-        # Mac mtimes need a bit of special casing
-        if mtime < 0x7fffffff:
-            mtime = int(mtime)
-        else:
-            mtime = int(-0x100000000 + int(mtime))
+    mtime = int(mtime)
```

Comment:
This and the next seem like different types of changes.

Reply:
This is a very old code, written when time could be int. It is complicated and not completely correct.

This change is included in #19708, so it will be removed from this PR.